schema/Comment.schema.yaml

```diff
@@ -0,0 +1,18 @@
+title: Comment
+'@id': schema:Comment
```

Comment:
The [`@id` on `CreativeWork`](https://github.com/stencila/schema/pull/228/files#diff-c684434113527c3fdd0ebed6e9accf1e46122039b97587612be7ec8231dc24c7R35) is lowercase, should this follow suit as well?